libs/blocks/table/table.js

```diff
@@ -140,10 +140,17 @@ function handleTitleText(cell) {
   if (cell.querySelector('.table-title-text')) return;
   const textSpan = createTag('span', { class: 'table-title-text' });
```

Comment:
I would suggest something such as
```js
const textSpan = createTag('span', { class: 'table-title-text' });
while (cell.firstChild) textSpan.append(cell.firstChild);

const iconTooltip = textSpan.querySelector('.icon-info, .icon-tooltip, .milo-tooltip');
if (iconTooltip) cell.append(iconTooltip.closest('em') || iconTooltip);

const firstIcon = textSpan.querySelector('.icon:first-child');
let nodeToInsert = textSpan;

if (firstIcon) {
  const titleRowSpan = createTag('span', { class: 'table-title-row' });
  titleRowSpan.append(firstIcon, textSpan);
  nodeToInsert = titleRowSpan;
}

cell.insertBefore(nodeToInsert, cell.firstChild);
```

Reply:
@mokimo , I changed the code.
Thank you for the suggestion; it now looks good.